### PR TITLE
Add Getting Started Guide for S3

### PR DESCRIPTION
**Pull request by HotSushi**
## Summary
This PR adds instructions for setting up s3 on docker. We use tabs with defaults so that navigation is easy.
Changes to the instructions are:
1. docker compose directory
2. spark-shell --jars 
3. spark-shell --confs

## Changes
- [X] Documentation

## Testing Done
```
npm install
npm run start
```
The outcome of the PR is captured in the screenshot:
image
image

### CODE REVIEW

File: docs/getting_started.mdx [hunk 1]
```diff
@@ -8,6 +8,75 @@ tags:
   - Iceberg
 sidebar_position: 2
 ---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+    <TabItem value="s3" label="S3" default>
+# OpenHouse on Spark & S3
```
Comment (jainlavina) [inline docs/getting_started.mdx:17]: s/on/with

File: docs/getting_started.mdx [hunk 2]
```diff
@@ -8,6 +8,75 @@ tags:
...
+
+In this guide, we will quickly set up a running environment and experiment with some simple SQL commands. Our
+environment will include all the core OpenHouse services such as [Catalog Service](./intro.md#catalog-service),
+[House Table service](./intro.md#house-table-service) and [others](./intro.md#control-plane-for-tables),
+[a Spark 3.1 engine](https://spark.apache.org/releases/spark-release-3-1-1.html) and
+also [MinIO S3 Instance](https://min.io/docs/minio/container/index.html). By the end of this walkthrough, we will have created some tables on OpenHouse,
```
Comment (jainlavina) [inline docs/getting_started.mdx:23]: In this walkthrough, we will create some tables on Openhouse, insert data in them and query the data.

File: docs/getting_started.mdx [hunk 3]
```diff
@@ -8,6 +8,75 @@ tags:
...
+--conf spark.sql.catalog.openhouse.metrics-reporter-impl=com.linkedin.openhouse.javaclient.OpenHouseMetricsReporter    \
+--conf spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080   \
+--conf spark.sql.catalog.openhouse.auth-token=$(cat /var/config/openhouse.token) \
+--conf spark.sql.catalog.openhouse.cluster=LocalS3Cluster
+```
+:::note
+the configuration `spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080` points to the docker container
+running the [OpenHouse Catalog Service](./intro.md#catalog-service).
+:::
+:::note
```
Comment (jainlavina) [inline docs/getting_started.mdx:73]: Also add a note about io-impl pointing to S3FileIO?

File: docs/getting_started.mdx [hunk 4]
```diff
@@ -8,6 +8,75 @@ tags:
...
+you can access the MinIO UI at `http://localhost:9871` of your host machine and inspect the state of objects
+created for your table. The username is `admin` and password is `password` for the MinIO docker setup.
+:::
+
+    </TabItem>
+    <TabItem value="hdfs" label="HDFS">
 # OpenHouse on Spark & HDFS
```
Comment (jainlavina) [inline docs/getting_started.mdx:80]: s/on/with

File: docs/getting_started.mdx (hunk 3 above)
Comment (HotSushi) [reply docs/getting_started.mdx:74]: PTAL

File: docs/getting_started.mdx (hunk 2 above)
Comment (HotSushi) [reply docs/getting_started.mdx]: thanks for the quick review!

**Review by jainlavina — APPROVED** [review]

**Review by sumedhsakdeo — APPROVED** [review]

**Pull request merged**